backend/server/database.py

```diff
@@ -112,7 +110,7 @@ def create_dynamic_db():
                             "isSummerEnabled": {
                                 'bsonType': 'bool'
                             },
-                            "plan": {
+                            "years": {
```

Comment:
the frontend has this as `plan`, I think we should have this as `plan` for consistency.

Reply:
Hmm as far as I can tell, it's `years`

(Have a look at plannerSlice.ts)

Reply:
Gabriel told me to get data from the root local storage instead of redux slices?  https://compclub.atlassian.net/jira/software/projects/CIRCLES/boards/5?selectedIssue=CIRCLES-346 

iirc last time i printed it out it has a different data structure

Reply:
Hmm I also printed it out, that's how I noticed the differences 

Can you double check on your end?

Reply:
oops my bad, yup, it's `years` sorry for that xD